src/main/java/com/google/devtools/build/lib/bazel/repository/RepositoryOptions.java

```diff
@@ -46,6 +46,21 @@ public class RepositoryOptions extends OptionsBase {
               + "otherwise the default of '<output_user_root>/cache/repos/v1' is used")
   public PathFragment experimentalRepositoryCache;
 
+  @Option(
+      name = "repo_contents_cache",
```

Comment:
nit: I kind of prefer `--repository_content_cache` to keep it close to the existing flag.

Reply:
added as an alias